core/templates/dev/head/pages/exploration_player/information_card_modal_directive.html

```diff
@@ -95,3 +96,10 @@ <h2 class="oppia-info-card-title"><[explorationTitle]></h2>
     <span class="oppia-icon-accessibility-label">Close</span>
   </button>
 </md-card>
+
+<style>
+  .private {
```

Comment:
We've talked about this before, I think -- this is too broadly scoped. Make sure that it applies only to the information card modal and not to anywhere around the application which uses the "private" class.

Reply:
I'll use the following argument : .oppia-info-card-content .private-card-info